### DISCUSSION TITLE
[APP] Make conditions selectable

### DISCUSSION THREAD

**Discussion by Attacler** [directus/directus#10730] [Feature Requests]
Hi there,

I think that it would be fancy if you can make a condition within a collection.
And then when you have other fields that would need the same condition, that you can select the first one.
So basicly conditions are linked to a collection (O2M) and you can add existing ones to a field of choice.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
